.github/workflows/build_ensnode_images.yml

```diff
@@ -0,0 +1,45 @@
+name: "Build: ENSNode Images"
+on:
+  workflow_dispatch:
+  release:
+    types: [published]
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  ensnode-build-and-push:
+    runs-on: blacksmith-4vcpu-ubuntu-2204
```

Comment:
I wonder if this CI Worker will have enough disk space to deal with the images to be built for ENSRainbow? 

I guess one way of figuring it out is to go live with this PR and see.